ml4h/tensor_generators.py

```diff
@@ -666,13 +666,6 @@ def get_train_valid_test_paths(
 
     logging.info(f'Found {len(train_paths)} train, {len(valid_paths)} validation, and {len(test_paths)} testing tensors at: {tensors}')
     logging.debug(f'Discarded {len(discard_paths)} tensors due to given ratios')
-    if len(train_paths) == 0 or len(valid_paths) == 0 or len(test_paths) == 0:
```

Comment:
can we just replace the `or` with `and` if there is no data we still want fail fast right?